### ISSUE TITLE
Set up Travis CI and update it to test up to PHP 8

### ISSUE THREAD

**Issue by Shardj** [Shardj/zf1-future#131]

**Comment by onlime** [comment]
I have committed #132 which is basically just a (probably not yet working) backport of Travis/unittest enhancements in https://github.com/zf1s/zf1

In onlime/zf1-future@a5446c2a2d1ca6b4eb05b70b12ed0b7e07c02208 have just applied zf1s/zf1@8eb04b2977e173ee614ca86761bc63057d4f2724 like this, quick and dirty:

```bash
$ wget https://github.com/zf1s/zf1/commit/8eb04b2977e173ee614ca86761bc63057d4f2724.patch -O zf1s-zf1-8eb04b.patch
$ sed -i 's#// require_once#require_once#g' zf1s-zf1-8eb04b.patch
$ perl -pi -e 's#packages/zend\-.*?/##g' zf1s-zf1-8eb04b.patch
$ patch -p1 < zf1s-zf1-8eb04b.patch
```

then backported `.travis.yml` and some fixes in onlime/zf1-future@30b510e0ec7c88b5fee0569b111f5aff94cd93f5

Please note, I have no experience in Travis at all. I am just into GitLab CI/CD infrastructure which I greatly prefer over Github.

@Shardj Hope you get this running!

**Comment by Shardj** [comment]
I didn't intend for this to be closed
